packages/cli/src/commands/type-check.js

```diff
@@ -57,11 +57,9 @@ export const handler = async ({ sides, verbose, prisma, generate }) => {
 
     const tscForAllSides = sides.map((side) => {
       const projectDir = path.join(getPaths().base, side)
-      // -s flag to suppress error output from yarn. For example yarn doc link on non-zero status.
-      // Since it'll be printed anyways after the whole execution.
       return {
         cwd: projectDir,
-        command: `yarn -s tsc --noEmit --skipLibCheck`,
+        command: `yarn tsc --noEmit --skipLibCheck`,
```

Comment:
@dac09 @callingmedic911 as far as I can tell, yarn 3 doesn't seem to have this flag anymore and I haven't found a replacement for it yet. Do you guys have any ideas?

Reply:
Yarn@>=2 doesn't print anything when you run scripts anymore so no reason for the flag to exist nor a replacement for it.